File: frontend2/src/app/shared/models/corporate-order.model.ts
import { PaymentMethod } from '../enums/payment-method.enum';
import { CorporateOrderProduct } from './corporate-order-product.model';

export class CorporateOrder {
  id: string;
  createdOn: string;
  number: string;
  cost: number;
  costMethod: PaymentMethod;
  prepayment: number;
  prepaymentMethod: PaymentMethod;
  deliveryCost: number;
  deliveryCostMethod: PaymentMethod;
  tax: number;
  profit: number;
  region: string;
  city: string;
  companyName: string;
  contactPerson: string;
  phone: string;
  email: string;
  notes: string;
  products: CorporateOrderProduct[];

  constructor(data: any) {
    Object.assign(this, data);
  }
}

File: Mushka.WebApi/ClientModels/CorporateOrder/GetById/CorporateOrderModel.cs
using System;
using Mushka.Domain.Entities;

namespace Mushka.WebApi.ClientModels.CorporateOrder.GetById
{
    public class CorporateOrderModel
    {
        public Guid Id { get; set; }

        public DateTime CreatedOn { get; set; }

        public string OrderNumber { get; set; }

        public decimal Cost { get; set; }

        public PaymentMethod CostMethod { get; set; }

        public decimal? Prepayment { get; set; }

        public PaymentMethod? PrepaymentMethod { get; set; }

        public decimal? DeliveryCost { get; set; }

        public PaymentMethod? DeliveryCostMethod { get; set; }

        public int? Tax { get; set; }

        public decimal Profit { get; set; }

        public string Region { get; set; }

        public string City { get; set; }

        public string CompanyName { get; set; }

        public string ContactPerson { get; set; }

        public string Phone { get; set; }

        public string Email { get; set; }

        public string Notes { get; set; }
    }
}